components/src/core/components/Dialog/Dialog.vue

```diff
@@ -11,6 +11,8 @@
           v-bind="$attrs"
           role="document"
           @click="onClickSheet"
+          v-focus-trap
```

Comment:
If we focus trap for dialog, we need to setup `esc` and/or `backspace` as ways to close Dialog.

`<oxd-overlay role="dialog" centered :show="true" @click="onClickOverlay" @keyup.esc="onClickOverlay" @keyup.delete="onClickOverlay">`

Reply:
Yes, this is something we will have to improve. I'll work on it.